src/app/routes/routes.ts

```diff
@@ -53,5 +54,9 @@ export const appRoutes: Routes = [
     {
         path: 'curriculum/dialog-view',
         component: DialogViewComponent
+    },
+    {
+        path: 'forgotten/password',
+        component: ForgottenPasswordComponent
```

Comment:
routes.ts doesn't exist anymore, put this code in routes.module

Reply:
Completed.